packages/create-plugin/templates/common/.config/webpack/BuildModeWebpackPlugin.ts

```diff
@@ -0,0 +1,34 @@
+import * as webpack from 'webpack';
+
+const PLUGIN_NAME = 'BuildModeWebpack';
+
+export class BuildModeWebpackPlugin {
+  apply(compiler: webpack.Compiler) {
+    compiler.hooks.compilation.tap(PLUGIN_NAME, (compilation) => {
+      compilation.hooks.processAssets.tap(
```

Comment:
Is this done as late as possible in the compilation process? Just thinking it would be good to only do this if the build seems to be successful.

Reply:
Yeah I think it's at a late state. Tested with a build that failed, and then this prop is not added to the asset.